### PR TITLE
feat: (nft): Nft activity section for collections and individual nfts

### PR DESCRIPTION
**Pull request by memoyil**
To review: 

https://deploy-preview-2666--pancakeswap-dev.netlify.app/

### CODE REVIEW

**Comment by roeerr** [comment]
Amazing!, Please add the 10 sec auto refresh and full filter with delist etc... and more history paging

**Comment by memoyil** [comment]
> Amazing!, Please add the 10 sec auto refresh and full filter with delist etc... and more history paging

Full filter list added you can check.

**Comment by roeerr** [comment]
> > Amazing!, Please add the 10 sec auto refresh and full filter with delist etc... and more history paging
> 
> Full filter list added you can check. 10 sec auto refresh might increase the load on graph ql server I tend not to do that.

Maybe refresh button?

**Comment by roeerr** [comment]
> To review:
> 
> https://deploy-preview-2666--pancakeswap-dev.netlify.app/

Can add price filter too?

**Comment by memoyil** [comment]
@roeerr Refresh button, more history paging and price filter added. You can check

**Comment by roeerr** [comment]
> @roeerr Refresh button, more history paging and price filter added. You can check

wow is more then amazing its working fantastic 👍 
I would really love for it to be approved!!

File: src/views/Nft/market/types/MarketEvent.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export enum MarketEvent {
```
Comment (Chef-Cheems) [inline src/views/Nft/market/types/MarketEvent.tsx:1]: Let's move this and Activity.tsx to `src/views/Nft/market/types.ts`

File: src/views/Nft/market/Profile/components/ActivityHistory/index.tsx [hunk 2]
```diff
@@ -28,9 +29,9 @@ const ActivityHistory = () => {
   const { t } = useTranslation()
   const [currentPage, setCurrentPage] = useState(1)
   const [maxPage, setMaxPages] = useState(1)
-  const [activitiesSlice, setActivitiesSlice] = useState<Activity[]>([])
+  const [activitySlice, setActivitySlice] = useState<Activity[]>([])
   const [nftMetadata, setNftMetadata] = useState<NftToken[]>([])
-  const [sortedUserActivities, setSortedUserActivities] = useState<Activity[]>([])
+  const [sortedUserActivity, setSortedUserActivity] = useState<Activity[]>([])
```
Comment (Chef-Cheems) [inline src/views/Nft/market/Profile/components/ActivityHistory/index.tsx:34]: Not sure why rename Activities -> Activity. It's an array so plural is more appropriate, no? 😄

File: src/views/Nft/market/types/MarketEvent.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Nft/market/types/MarketEvent.tsx]: I couldn't move there it gave me dependency cycle error. I moved to `state/nftMarket/types.ts` instead

File: src/views/Nft/market/Profile/components/ActivityHistory/index.tsx (hunk 2 above)
Comment (memoyil) [reply src/views/Nft/market/Profile/components/ActivityHistory/index.tsx]: Done

**Review by Chef-Cheems — APPROVED** [review]

**Pull request merged**